CodeEdit/NavigatorSidebar/SourceControlNavigator/SourceControlNavigatorView.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  SourceControlNavigatorView.swift
+//  CodeEdit
+//
+//  Created by Nanashi Li on 2022/05/05.
+//
+
+import SwiftUI
+import CodeEditUI
+
+struct SourceControlNavigatorView: View {
+
+    @ObservedObject
+    private var workspace: WorkspaceDocument
```

Comment:
nit: this can be a constant

Reply:
marked as unresolved since this wasn't changed

Reply:
Property wrapper can only be applied to a 'var'.... Please do research on Property Wrappers